api/envoy/extensions/filters/network/mongo_proxy/v3/mongo_proxy.proto

```diff
@@ -39,4 +40,9 @@ message MongoProxy {
   // Flag to specify whether :ref:`dynamic metadata
   // <config_network_filters_mongo_proxy_dynamic_metadata>` should be emitted. Defaults to false.
   bool emit_dynamic_metadata = 4;
+
+  // List of commands to emit metrics for. Defaults to "delete", "insert", and "update".
+  // Note that metrics will not be emitted for "find" commands, since those are considered
+  // queries, and metrics for those are emitted under a dedicated "query" namespace.
+  repeated string commands = 5;
```

Comment:
Please add a release note.

Reply:
Done.

Reply:
I don't see a release note? Please also ref link to this field. You will need to merge main to pick up the CI change for docs also.

/wait

Reply:
Sorry, should be there now.